src/app/vocabularyTypes.json

```diff
@@ -4,27 +4,31 @@
     "prefix": "https://slovník.gov.cz/datový/",
     "regex": "^https://slovník.gov.cz/datový/[ěščřžýáíéóúůďťňa-z0-9-]+$",
     "autoIriRegex": "[ěščřžýáíéóúůďťňa-z0-9]+",
+    "autoIriMatchAll": true,
     "level": 4
   },
   {
     "label": "Agendový",
     "prefix": "https://slovník.gov.cz/agendový/",
     "regex": "^https://slovník.gov.cz/agendový/[a-z0-9-]+$",
```

Comment:
If we relax agenda not to be necessarily bound to RPP, the we should allow Czech characters similarly to the generic vocabularies IMO.

Reply:
Also, assembly-line-shared utils contain the regexes - https://github.com/opendata-mvcr/assembly-line-shared/blob/main/src/vocabulary-utils/vocabularyTypes.json - should be reused.

Reply:
OK, I see two problems here.

First the structure of SGoV is not really clear.

Second, regarding the reuse of regexes - potentially doable, but right now there is very little overlap between the vocabulary name related features in MC and in the shared library. If we move the configuration to the library, it may contain MC-specific stuff. Alternatively we can reuse only a part of the library-based configuration in MC, but that smells like trouble to me.